src/main/Body.tsx

```diff
@@ -39,6 +40,7 @@ const Body: React.FC<{}> = () => {
         <div css={bodyStyle}>
           <MainMenu />
           <MainContent />
+          <Lock />
```

Comment:
Since this doesn't render anything, it could probably go even higher in the hierarchy.